public/ng-views/maps.html

```diff
@@ -19,23 +22,23 @@
         <small>{{ user.name }}</small>
       </i>
     </span>
-  </span>
+  </span> -->
 </div>
 
 
-<div class="topnav right">
+<!-- <div class="topnav right">
   <div class="fa-stack" ng-click="saveCombo=true">
     <i class="fa fa-circle fa-stack-2x"></i>
     <i class="fa fa-arrow-left fa-stack-1x fa-inverse"></i>
-  </div>
-  <div class="fa-stack fa-lg" ng-click="dotSelect = !dotSelect; updateTargetFeature(); overview=false; tabnav=null">
+  </div> -->
+  <!-- <div class="fa-stack fa-lg" ng-click="dotSelect = !dotSelect; updateTargetFeature(); overview=false; tabnav=null">
```

Comment:
Are you commenting out angular code? Please remove unnecessary code... It makes it very hard to read. Also I believe just commenting out angular code is dangerous - it also just turns everything to spaghetti - extremely hard to parse.